.gitignore

```diff
@@ -9,3 +9,4 @@ pom.xml.asc
 /.nrepl-port
 .cljs_rhino_repl
 node_modules
+*.swp
```

Comment:
I generally don't like to pollute repo-specific .gitignores with editor or tool-specific items. This can go in a global gitignore